server/routes/prisoner.ts

```diff
@@ -42,8 +42,11 @@ export default function routes({ auditService, prisonerProfileService }: Service
   post('/:offenderNo', async (req, res) => {
     const offenderNo = getOffenderNo(req)
     const { prisonId } = req.session.selectedEstablishment
+    const { username } = res.locals.user
 
-    const { prisonerDetails } = await prisonerProfileService.getProfile(prisonId, offenderNo, res.locals.user.username)
+    const { prisonerDetails, activeAlerts } = await prisonerProfileService.getProfile(prisonId, offenderNo, username)
+
+    const restrictions = await prisonerProfileService.getRestrictions(offenderNo, username)
```

Comment:
```suggestion
const [{ prisonerDetails, activeAlerts }, restrictions] = await Promise.all([
      prisonerProfileService.getProfile(prisonId, offenderNo, username),
      prisonerProfileService.getRestrictions(offenderNo, username),
    ])
```

These calls are independent so no need for them to happen sequentially. `Promise.all()` will fire them off together to make it quicker.